tools/ops/policystream.py

```diff
@@ -296,21 +310,25 @@ def initialize_tree(self, tree):
                 yaml.safe_load(self.repo.get(tree[fpath].id).data),
                 Config.empty(), fpath)
 
+    def _get_policy_fents(self, tree):
+        # get policy file entries from a tree recursively
+        results = {}
+        q = deque([tree])
+        while q:
+            t = q.popleft()
+            for fent in t:
```

Comment:
Since there are no tests around this just wanted to warn you that this threw an error for me when I tried this code. '_pygit2.TreeEntry' object is not iterable

Reply:
ugh.. k, thanks for the heads up. the logic in total on this script is complicated enough that it would benefits from tests but it might be better for this to be in a pkg before thats done to line up with dependency install and existing test execution.  fwiw, i've been running it on a few hundred repos with a few years of history, and 10k+ policies, and it works well, albeit i don't think any of our repos have subdirs for policies, i'll take a look at constructing something like that prior to merging.

Reply:
i've converted over to a package and added some unit tests and a dockerfile